src/accelerate/accelerator.py

```diff
@@ -511,6 +524,22 @@ def local_process_index(self):
     def device(self):
         return self.state.device
 
+    @property
+    def split_batches(self):
+        return self.dataloader_config.split_batches
+
+    @property
+    def dispatch_batches(self):
+        return self.dataloader_config.dispatch_batches
+
+    @property
+    def even_batches(self):
```

Comment:
This suggestion would currently fail:

https://github.com/huggingface/accelerate/blob/613ad7089a4343341c1d89ab96998703b4dbf65c/src/accelerate/data_loader.py#L158

So either the suggestion could be adjusted, or we should have setters for these properties (which would be better for BC overall).